response_operations_ui/controllers/sign_in_controller.py

```diff
@@ -0,0 +1,17 @@
+import json
+import logging
+
+import requests
+from structlog import wrap_logger
+
+from response_operations_ui import app
+
+logger = wrap_logger(logging.getLogger(__name__))
+
+
+def get_sign_in(sign_in_data):
+    logger.debug('Retrieving sign in authorisation')
```

Comment:
I don't think this is the correct name for the function. Please rename to sign_in or something similar